src/mui/list/InfiniteList.js

```diff
@@ -99,7 +99,9 @@ export class InfiniteList extends List {
         const resourceName = translate(`resources.${resource}.name`, {
             smart_count: 2,
             _: inflection.humanize(inflection.pluralize(resource)),
-        });
+        })
+            .toString()
+            .toLowerCase();
```

Comment:
Why lower case?

Reply:
I added the lowercase to remove the capital letter from the resource name because we get something like this otherwise => "Liste des Factures" and now we get => "Liste des factures"

Reply:
Ah ok, I'm in English on my browser so I have exactly the reciprocal issue :)
(I mean "invoices List")